### ISSUE TITLE
Cannot get Readme size via octokit.net

### ISSUE THREAD

**Issue by qweeah** [octokit/octokit.net#2233]
Need to access the `size` of README.md file, only to find that it's not a public property of Readme object. 

The property itself is indeed available  in native [Github API](https://docs.github.com/en/rest/reference/repos#get-a-repository-readme). Better to have it in octokit.net.

**Comment by shiftkey** [comment]
I think this should be resolved by:

 - Adding `Size` as a property on `Octokit/Models/Response/ReadmeResponse.cs`
 - Adding `Size` as a property on `Octokit/Models/Response/Readme.cs` and assigning that from the `response`

Marking this as up-for-grabs as it's an ideal small contribution

**Comment by littlemisslilycane** [comment]
I worked with Octokit library in the past and wanted to contribute. This is my first open-source contribution. 
I have added the Size fields to the Readme.cs & ReadmeResponse.cs. Modified the test cases to include the new parameter. I tested it locally. 
Quick question on adding new fields to a class (and constructor). I added the Size field after the Name field based on the actual API response format. Is that the correct practice to add a new field or any new fields should be added at the last? Thank you so much!
